src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/CaseRetrievalTest.kt

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.controllers
+
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Import
+import org.springframework.context.annotation.Primary
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import uk.gov.hmcts.reform.bulkscan.orchestrator.Application
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.IMessageReceiverProvider
+import java.util.concurrent.CountDownLatch
+
+val latch = CountDownLatch(1)
+
+@Import(Application::class)
+open class TestConfig {
```

Comment:
I think if you apply the kotlin-spring compiler plugin this doesn't need to be open
https://kotlinlang.org/docs/tutorials/spring-boot-restful.html
https://kotlinlang.org/docs/reference/compiler-plugins.html#kotlin-spring-compiler-plugin

Reply:
sure ... we can improve :D

Reply:
done